spec/lib/triannon-client/triannon-client_spec.rb

```diff
@@ -91,86 +177,127 @@ def check_graph_has_statements(graph)
     end
     context 'with no content_type' do
       it 'raises an argument error with a nil ID' do
-        expect{@tc.get_annotation(nil)}.to raise_error(ArgumentError)
+        tc = TriannonClient::TriannonClient.new
```

Comment:
It was all in a before each block, but add the respond_to method checks was crashing all the tests; somehow rspec was auto-stubbing an @tc object and that infered with the before setup somehow.  But, yes, I do want to find a way to do this.  I'm not entirely comfortable with `let` because it does something like `@tc || TriannonClient::TriannonClient.new` and so it will always use the same client and the respond_to checks could mess it up (don't want to go down that road again).